### DISCUSSION TITLE
Follow semantic versioning

### DISCUSSION THREAD

**Discussion by kkiermasz** [airbnb/lottie-ios#1782] [General]
Hey there!

Since the Lottie supports an SPM I think it's safe to say that we should follow its rules. One of them relates to the versioning of libraries. [In the documentation](https://developer.apple.com/documentation/packagedescription/version) we can find below quote:

> A package version consists of three integers separated by periods, for example 1.0.0. It must conform to the semantic versioning standard in order to ensure that your package behaves in a predictable manner once developers update their package dependency to a newer version.

Latest release (3.5.0) includes #1420, which is not backward-compatible.

I assume that the dangers of such a release are clear, but if you need more information regarding this, just let me know :)
Because of the above, I just wanted to start the discussion about sticking to semver convention.

**Comment by calda** [comment]
Lottie 3.5.0 doesn't include any source breaking changes for Swift code, so complies with semver. We are saving the source-breaking changes for Lottie 4.0, which hasn't been released yet.

**Reply by calda** [reply]
There was one breaking change in Lottie 3.5.0, which was unfortunate but not intentional: https://github.com/airbnb/lottie-ios/issues/1771. We didn't discover this issue until after Lottie 3.5.0 had been released, but it seems relatively minor to me. Are you referring to this issue, or something else?

**Reply by kkiermasz** [reply]
Gosh, I'm really sorry. While I was OOO one of my colleagues made some changes to our codebase. He used some staff from 3.5.0 without changing the required version in the package manifest. 
Thus the only thing that is wrong related to the 3.5.0 release is me :D
Anyway, thanks for such a fast reaction! Have a nice day
